tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -1,4 +1,13 @@
 jobs:
+    steps:
```

Comment:
prefer to keep changes outside of the scope of ACL changes in a separate PR.

Reply:
I have removed ACL changes and will create a new PR for that.